byebug.gemspec

```diff
@@ -25,5 +25,7 @@ Gem::Specification.new do |s|
   s.extensions = ["ext/byebug/extconf.rb"]
   s.require_path = "lib"
 
+  s.add_dependency "fiddle", "~> 1.0"
```

Comment:
It definitely feels kind of sad to be taking `byebug` from being a dependency-free gem to now having a runtime dependency, but I guess that unfortunately that might sort of be the inevitable result for some gems of Ruby moving more and more of its non-core functionality out of the standard library and into gems.

Reply:
I wasn't sure which version specifier to put here. I went with `~> 1.0` because:

1. `fiddle` v1.0.0 was released quite a while ago, on 2017-12-06 ([source](https://rubygems.org/gems/fiddle/versions)).
2. That ensures that we will require the latest `fiddle` major version (i.e. version 1) and that we won't pull in any subsequent major version (e.g. version 2) that might include breaking changes.